packages/macro/test/js-defineMessage.ts

```diff
@@ -36,19 +36,22 @@ export default [
     `,
   },
   {
-    name: "should left string message intact - template literal",
+    name: "should transform template literals",
```

Comment:
Normally thiw would be a breaking change, but since the code didn't work as described in documentation, I believe it's just a fix ;)

Reply:
I hope so 🤪